tests/core/arch-arm64/stalker-arm64.c

```diff
@@ -346,6 +352,142 @@ store_x0 (GumCpuContext * cpu_context,
   *last_x0 = cpu_context->x[0];
 }
 
+STALKER_TESTCASE (exclude_bl)
+{
+  const guint32 code_template[] =
+  {
+    0xa9bf7bf3, /* push {x19, lr} */
+    0xd2801553, /* mov x19, #0xaa */
+    0xd2800883, /* mov x3, #0x44  */
+    0xd2800662, /* mov x2, #0x33  */
+    0xd2800441, /* mov x1, #0x22  */
+    0xd2800220, /* mov x0, #0x11  */
+    0xa9bf07e0, /* push {x0, x1}  */
+    0x94000009, /* bl func_a      */
+    0xa8c107e0, /* pop {x0, x1}   */
+    0xd2801103, /* mov x3, #0x88  */
+    0xd2800ee2, /* mov x2, #0x77  */
+    0xd2800cc1, /* mov x1, #0x66  */
+    0xd2800aa0, /* mov x0, #0x55  */
+    0x94000005, /* bl func_b      */
+    0xa8c17bf3, /* pop {x19, lr}  */
+    0xd65f03c0, /* ret            */
+
+    /* func_a: */
+    0xd2801100, /* mov x0, #0x88  */
+    0xd65f03c0, /* ret            */
+
+    /* func_b: */
+    0xd2801320, /* mov x0, #0x99  */
+    0xd65f03c0, /* ret            */
+  };
+
+  StalkerTestFunc func;
+  guint8 * func_a_address;
+  guint64 counter = 0;
+
+  fixture->transformer = gum_stalker_transformer_make_from_callback (
+      count_instructions, &counter, NULL);
+
+  g_assert_cmpuint (counter, ==, 0);
+
+  func = GUM_POINTER_TO_FUNCPTR (StalkerTestFunc,
+      test_arm64_stalker_fixture_dup_code (fixture, code_template,
+        sizeof (code_template)));
+
+  func_a_address = fixture->code + (16 * 4);
+
+  GumMemoryRange memory_range;
+  memory_range.base_address = (GumAddress) func_a_address;
+  memory_range.size = (4 * 2);
+
+  gum_stalker_exclude (fixture->stalker, &memory_range);
+
+  test_arm64_stalker_fixture_follow_and_invoke (fixture, func, 0);
+
+  g_assert_cmpuint (counter, ==, 24);
+}
+
+static void
+count_instructions (GumStalkerIterator * iterator,
+                    GumStalkerWriter * output,
+                    gpointer user_data)
+{
+  guint64 * counter = user_data;
+  const cs_insn * insn;
+
+  while (gum_stalker_iterator_next (iterator, &insn))
+  {
+    *counter = *counter + 1;
+    gum_stalker_iterator_keep (iterator);
+  }
+}
+
+static void
+hello_ken (void)
+{
+  g_print ("hello ken! ");
```

Comment:
Tests shouldn't print anything unless it's really needed. Perhaps just a `g_usleep (1)` instead?

Reply:
I was initially thinking about `g_usleep (1)`, but I wanted an immediate feedback that the function is actually invoked. 
Now I see no reason why the function should not be called. I'm changing it.